lib/openai/client.rb

```diff
@@ -17,7 +18,10 @@ def initialize(config = {}, &faraday_middleware)
       CONFIG_KEYS.each do |key|
         # Set instance variables like api_type & access_token. Fall back to global config
         # if not present.
-        instance_variable_set("@#{key}", config[key] || OpenAI.configuration.send(key))
```

Comment:
Turns out the statement `config[key] || OpenAI.configuration.send(key)` will always return the right hand side if the left hand side is `false`, which I don't think is the intention here.

Reply:
eep good catch.